src/process/releases.md

```diff
@@ -3,3 +3,56 @@ title: Releases
 type: process
 order: 510
 ---
+
+We aim to follow the Gitflow strategy of deploying sites in combination with CI/CD.
+
+## Steps
+
+To deploy a site of system, you can roughly follow these steps:
+
+1. In Jira, Identify tickets ready to release
+    a. Issues
```

Comment:
I imagine this is in reference to the issue dropdown in the menu. However this might not be clear to all.